float64 z_act
float64 w_act
float64 pitch_act
float64 q_act
float64 delta_v_act
float64 delta_s_act
float64 delta_m_act
float64 zend
float64 thetaend
float64 tf
float64 u_act
float64 t0
float64 tend
float64 yaw_act

